changelog/LATEST.md

```diff
@@ -31,10 +72,42 @@ View all [changelogs](https://github.com/Divine-Journey-2/Divine-Journey-2/tree/
 - Updates loading screen tip to include the information that Erebus is skippable.
 - Updates and improve some tips.
 - Updates Upgrade Kits quest to display upgrades and number of Augments more accurately.
+- Changes the Spores Reactor quest dependency from Plastic to Fertilizer, since that is what it costs.
+- Adds a tooltip to the Aether Legacy Altar to inform about the "sped up while above Gravitite Block" feature.
+- Adds information to the Core Sample Drill Quest that Core Samples can be placed down in-world.
+- Adds information about the Mekanism Gauge Dropper to the PRC Quest, and adds a tooltip about being able to clear it via Shift+Right Click.
+- Fixes a typo in the Mana Spreader Lens quest.
+- Changes the dependency of the Roots Unendling Bowl to be off of Dark Steel (its cost), and adjust Chapter 10 accordingly.
+- Changes the dependency of the Astral Sorcery Attunement Altar to be off the Celestial Altar (what its crafted in), and adjust Chapter 23 accordingly.
+- Adds information about effect of Starlight to the production of the Astral Sorcery Lightwell to the Lightwell Quest.
+- Adds information about the stats of the Astral Sorcery Collector Crystal to the production of Starlight to the Celestial Collector Crystal Quest.
+- Increases the quest reward for obtaining Jetpacks, and attach the JetPlate Quests to the preceeding Jetpack Quests.
+- Updates the Galacticraft Communications Dish Card Quest to indicate the interaction must be with the base of the Communications Dish.
+- Fixes a random double-quote appearing in a tip.
+- Fixes a typo in the name of the Essence of Continuity.
+- Fixes missinformation about Stellar Conduits being enough to power the Weak Fusion Plant, suggest Cryo-Stabilized Fluxducts instead.
+- Fixes a typo where "rename" was incorrectly changed to "redesc" due to renaming lang keys from `x.name=` into `x.desc=`.
+- Fixes an incorrect statement about the logic of the quest being OR when it is AND.
+- Fixes an incorrectly possisive "it's" instead of "its".
+- Fixes Drawer tooltips incorrectly calling any potion "Witches' Brew", should now display the correct name.
+- Updates the Meteorite Compass Quest to state that it points to the Sky Stone Chest, due to a change while updating AE2-EL.
+- Notes that DivineRPG Furnaces cannot be automatically extracted from.
```

Comment:
Might want to specify that the output slots (and inputs) can not be extracted from, the fuel slot if the furnace has one does get extracted from (why).

Reply:
they cannot be automated. the specific way it doesnt work properly isnt worth noting